src/document/comment.ml

```diff
@@ -254,8 +254,10 @@ let tag : Comment.tag -> Description.one =
   | `Param (name, content) ->
       let value = Inline.Text name in
       item ~tag:"parameter" ~value (nestable_block_element_list content)
-  | `Raise (name, content) ->
-      let value = Inline.Text name in
+  | `Raise (kind, content) ->
+      let value =
+        Inline.Nested (inline_element (kind :> Comment.inline_element))
```

Comment:
I'm not a fan of this new `Nested` constructor. It can be avoided completely if `~value` is changed to be an `Inline.t`:

```patch
diff --git a/src/document/comment.ml b/src/document/comment.ml
index b0709b421..be0ff78d2 100644
--- a/src/document/comment.ml
+++ b/src/document/comment.ml
@@ -274,14 +274,11 @@ let tag : Comment.tag -> Description.one =
   let sp = inline (Text " ") in
   let item ?value ~tag definition =
     let tag_name = inline ~attr:[ "at-tag" ] (Text tag) in
-    let tag_value =
-      match value with
-      | None -> []
-      | Some t -> [ sp; inline ~attr:[ "value" ] t ]
-    in
+    let tag_value = match value with None -> [] | Some t -> sp :: t in
     let key = tag_name :: tag_value in
     { Description.attr = [ tag ]; key; definition }
   in
+  let mk_value desc = [ inline ~attr:[ "value" ] desc ] in
   let text_def s = [ block (Block.Inline [ inline @@ Text s ]) ] in
   let content_to_inline ?(prefix = []) content =
     match content with
@@ -293,25 +290,23 @@ let tag : Comment.tag -> Description.one =
   | `Deprecated content ->
       item ~tag:"deprecated" (nestable_block_element_list content)
   | `Param (name, content) ->
-      let value = Inline.Text name in
+      let value = mk_value (Inline.Text name) in
       item ~tag:"parameter" ~value (nestable_block_element_list content)
   | `Raise (kind, content) ->
-      let value =
-        Inline.Nested (inline_element (kind :> Comment.inline_element))
-      in
+      let value = inline_element (kind :> Comment.inline_element) in
       item ~tag:"raises" ~value (nestable_block_element_list content)
   | `Return content -> item ~tag:"returns" (nestable_block_element_list content)
   | `See (kind, target, content) ->
       let value =
         match kind with
-        | `Url -> Inline.Link (target, [ inline @@ Text target ])
-        | `File -> Inline.Source (source_of_code target)
-        | `Document -> Inline.Text target
+        | `Url -> mk_value (Inline.Link (target, [ inline @@ Text target ]))
+        | `File -> mk_value (Inline.Source (source_of_code target))
+        | `Document -> mk_value (Inline.Text target)
       in
       item ~tag:"see" ~value (nestable_block_element_list content)
   | `Since s -> item ~tag:"since" (text_def s)
   | `Before (version, content) ->
-      let value = Inline.Text version in
+      let value = mk_value (Inline.Text version) in
       item ~tag:"before" ~value (nestable_block_element_list content)
   | `Version s -> item ~tag:"version" (text_def s)
   | `Alert ("deprecated", content) ->
```

This also remove the `<span class="value">` around the reference, which doesn't seem useful.

Reply:
I've reverted the `Nested` constructor now and applied your patch. I think in the git history I had something similar at one point but then wanted to be consistent and have the `value` attribute on everything.